src/main/webapp/components/Export.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import * as FileSaver from 'file-saver';
+import * as XLSX from 'xlsx';
+import { formatExcelData } from "../util/TableUtil";
+
+export const Export = ({csvData, columns, fileName}) => {
+
+  const fileType = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;charset=UTF-8';
+  const fileExtension = '.xlsx';
```

Comment:
nit: can this be passed in as a prop? If feels forced that there can be no other extension, but the library might not be that friendly to non `xlsx` types.